pkg/master/master.go

```diff
@@ -784,12 +785,14 @@ func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
 	controllerStorage := expcontrolleretcd.NewStorage(c.ExpDatabaseStorage)
```

Comment:
This doesn't create a scale object I don't think. This creates a scale subpath on certain resources e.g.:

/api/v1/namespace/foo/replicationcontroller/bar/scale